test/standalone/Vegetation/plant_hydraulics_test.jl

```diff
@@ -335,7 +332,9 @@ for FT in (Float32, Float64)
             soln = nlsolve(
                 initial_compute_exp_tendency!,
                 Vector{FT}(-0.03:0.01:0.07);
-                ftol = 1e-11,
+                ftol = eps(FT),
+                method = :newton,
+                iterations = 20,
```

Comment:
I am not sure this is relevant but are 20 iterations enough to converge everywhere? Perhaps some places would take more iterations?

Reply:
good question! In this case, we are just running the test at one location/in a column, so it doenst matter, but yes, some places on the globe might take longer to converge. That said, I think many models still cap the number of iterations in places like this so that the entire run is not held up by a single point.